templates/buy-me-a-coffee-app/src/features/BuyMeACoffee/components/FormBuyCoffee.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState, useCallback } from 'react';
+
+import { useWaitForTransaction, usePrepareContractWrite, useContractWrite } from 'wagmi';
+
+import { parseEther } from 'viem';
+import { Card, Text, Flex, Box, Button, Heading, TextFieldInput, TextArea } from '@radix-ui/themes';
+
+import { contractAddress, contractABI } from '../contract/contractInfo';
+
+type FormBuyCoffeeProps = {
+  onComplete: () => void;
+};
+
+function FormBuyCoffee({ onComplete }: FormBuyCoffeeProps) {
```

Comment:
This is where the magic happen. I wonder if we should create a semi-standard around the location of this code, and reuse that standard across all our templates.

A bit like saying, you met on template you met them all.